web/src/controller/note_controller.rs

```diff
@@ -48,6 +48,41 @@ pub async fn create(
     Ok(Json(ApiResponse::new(StatusCode::CREATED.into(), note)))
 }
 
+#[utoipa::path(
+    put,
+    path = "/notes/{id}",
+    request_body(content = entity_api::notes::Model, content_type = "application/json"),
+    params(
+        ApiVersion,
+        ("id" = Id, Path, description = "Id of note to update"),
+    ),
```

Comment:
```suggestion
    params(
        ApiVersion,
        ("id" = Id, Path, description = "Id of note to update"),
    ),
    request_body = entity::notes::Model,
```

This change should enable RAPIdoc to display a default Note instance as JSON in the web interface instead of displaying "#/components/schemas/entity_api.notes.Model". For reference, the organization_controller does this with its update method.

Reply:
Thank you! I was wondering how to get that to work 
 08905d4d0362dca0acf6d3efc0074922ae002732